ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -175,13 +175,17 @@ int mca_pml_ucx_close(void)
 
 int mca_pml_ucx_init(void)
 {
+    ucp_worker_params_t params;
     ucs_status_t status;
     int rc;
 
     PML_UCX_VERBOSE(1, "mca_pml_ucx_init");
 
     /* TODO check MPI thread mode */
-    status = ucp_worker_create(ompi_pml_ucx.ucp_context, UCS_THREAD_MODE_SINGLE,
+    params.field_mask  = UCP_WORKER_PARAM_FIELD_THREAD_MODE;
+    params.thread_mode = UCS_THREAD_MODE_SINGLE;
```

Comment:
@alinask - shall we auto-detect it based on MPI_Init or MPI_Init_thread was called?